backend/kernelCI_app/views/hardwareDetailsView.py

```diff
@@ -101,24 +118,26 @@ def generate_test_dict():
     }
 
 
+@method_decorator(csrf_exempt, name='dispatch')
```

Comment:
add a TODO to remove this, we need to figure out a way to create post without making vulnerabilities, even though they are harmless in this particular example